src/app/components/address-input/address-input.component.html

```diff
@@ -23,8 +23,10 @@
                     trackBy: trackByFn
                 "
                 [value]="option.address"
+                class="contact-option"
             >
-                {{ option.label }} ({{ option.address }})
+                <span>{{ option.label }}</span
+                ><span>{{ option.address }}</span>
```

Comment:
minor stylistic thing the closing tag should not end on the next line

Reply:
Oh true. I should not completely rely on prettier

Reply:
I am not sure if this is prettier or tslint, (probably tslint) but I remember this driving me crazy. I think in the light client we solved it with an eslint configuration.